db/models.py

```diff
@@ -50,3 +54,80 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    bio = models.TextField(blank=True, null=True)
+
+    def __str__(self) -> str:
+        return self.username
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(default=now)
+    user = models.ForeignKey(
+        to=User,
+        on_delete=models.CASCADE,
+        related_name="orders"
+    )
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+    class Meta:
+        ordering = ("-created_at",)
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        to=MovieSession,
+        on_delete=models.CASCADE,
+        related_name="tickets"
+    )
+    order = models.ForeignKey(
+        to=Order,
+        on_delete=models.CASCADE,
+        related_name="tickets"
+    )
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    def __str__(self) -> str:
+        return f"{self.movie_session} (row: {self.row}, seat: {self.seat})"
+
+    def clean(self) -> None:
+        if self.row < 1 or self.row > self.movie_session.cinema_hall.rows:
+            raise ValidationError(
+                {
+                    "row": (
+                        f"row number must be "
+                        f"in available range: (1, rows): "
+                        f"(1, {self.movie_session.cinema_hall.rows})"
+                    )
+                }
+            )
+        if (
+            self.seat < 1
+            or self.seat > self.movie_session.cinema_hall.seats_in_row
+        ):
+            raise ValidationError(
+                {
+                    "seat": (
+                        f"seat number must be in "
+                        f"available range: (1, seats_in_row): "
+                        f"(1, {self.movie_session.cinema_hall.seats_in_row})"
+                    )
+                }
+            )
+
+    def save(self, *args, **kwargs) -> None:
+        self.full_clean()
```

Comment:
Calling `full_clean()` in the `save` method ensures that the `Ticket` instance is validated before saving, which is a good practice for maintaining data integrity.